README.md

```diff
@@ -44,7 +44,8 @@ By default, ld-scheduler polls launch darkly every 60 seconds:
     });
     ```
 
-3. Tag your feature flag. In launch darkly's dashboard, under the Settings tab of your feature flag, under Tags, add a tag called "scheduled". 
+3. Tag your feature flag. In launch darkly's dashboard, under the Settings tab of your feature flag, under Tags, add one or more "scheduled" tags, each prefixed with an environment name.
```

Comment:
I have updated a new screenshot to reflect the tag changes. The new image link is [here](https://cloud.githubusercontent.com/assets/1593077/23781221/ddd4557e-059f-11e7-8e4f-601e6cf91f78.png)